Registry.toml

```
@@ -98,6 +98,11 @@ name = "DiffEqParamEstim"
 location = "https://docs.sciml.ai"
 method = "hosted"
 
+[daee34ce-89f3-4625-b898-19384cb65244]
+name = "DocumenterCitations"
+location = "https://juliadocs.org/DocumenterCitations.jl/stable/"
```

Should this be
```suggestion
location = "https://juliaearth.github.io/GeoStatsDocs/stable/"
```
instead?